src/pendulum/parsing/__init__.py

```diff
@@ -214,11 +214,11 @@ def _parse_iso8601_interval(text: str) -> _Interval:
     first, last = text.split("/")
     start = end = duration = None
 
-    if first[0] == "P":
+    if first[0:1] == "P":
```

Comment:
How does that change anything? AFAIK `[0]` is the same as `[0:1]`.

Reply:
It's not the same. `[0:1]`, which could also be written as `[:1]`, is a lenient version of `[0]` -- it prevents from raising an `IndexError` on an empty string and instead evaluates to `''` in that case.